lib/imapoptions

```diff
@@ -2919,6 +2919,11 @@ of the incoming network interface, or if no record is found, the
 { "fastmailsharing", 0, SWITCH, "3.0.0" }
 /* If enabled, use FastMail style sharing (oldschool full server paths) */
 
+{ "normalizeuid", 0, SWITCH }
+/* Lowercase uid and strip leading and trailing blanks. It is recommended
+   to set this to yes, especially if OpenLDAP is used as authentication
+   source. */
+
```

Comment:
This "it is recommended" advice directly contradicts the comment around line 165 of auth_unix.c, where it explains that we used to do something naive like this, but then did it better instead...

Also, we already have a standard option that does the same thing, and is enabled by default:

>           username_tolower: 1
>               Convert  usernames to all lowercase before login/authentication.  This is
>               useful with authentication backends which  ignore  case  during  username
>               lookups (such as LDAP).

It doesn't trim leading and trailing whitespace... but if this is a real problem, we should apply a real fix (with tests).